From db65e07dee5846bafa974cd7ac611c034caee152 Mon Sep 17 00:00:00 2001
From: Tom Keddie <tomk@minim.com>
Date: Mon, 10 Apr 2023 07:39:07 -0700
Subject: [PATCH] linux: update 5.4 kernel to 5.4.238

---
 include/kernel-version.mk | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/include/kernel-version.mk b/include/kernel-version.mk
index 787abfca14..0ff4b7d436 100644
--- a/include/kernel-version.mk
+++ b/include/kernel-version.mk
@@ -1,4 +1,3 @@
-
 # Use the default kernel version if the Makefile doesn't override it
 LINUX_RELEASE?=1
 
@@ -9,13 +8,13 @@ endif
 LINUX_VERSION-3.18 = .29
 LINUX_VERSION-4.4 = .60
 LINUX_VERSION-4.14 = .193
-LINUX_VERSION-5.4 = .158
+LINUX_VERSION-5.4 = .238
 LINUX_VERSION-5.10 = .27
 
 LINUX_KERNEL_HASH-3.18.29 = d099072fcbd6aefea8c787c0325eaa56c65b35b15c52579c8a613349bcba425f
 LINUX_KERNEL_HASH-4.4.60 = e7f2f47acf17497d6ffd713eda65c025b3df0bce09faa8c04712bf1b3cfc9fdb
 LINUX_KERNEL_HASH-4.14.193 = 0b0fb41d4430e1a42738b341cbfd2f41951aa5cd02acabbd53f076119c8b9f03
-LINUX_KERNEL_HASH-5.4.158 = 6e018fecdc8fc24553756e582d83b82d65b10a6b03ef36262a24911f839b8d59
+LINUX_KERNEL_HASH-5.4.238 = 70a2b2da85598eba6a73cdc0749e441cbdf3011d9babcb7028a46aa8d98aa91f
 LINUX_KERNEL_HASH-5.10.27 = d99dc9662951299c53a0a8d8c8d0a72a16ff861d20e927c0f9b14f63282d69d9
 
 remove_uri_prefix=$(subst git://,,$(subst http://,,$(subst https://,,$(1))))
-- 
2.25.1

